### DISCUSSION TITLE
Calendar not syncing

### DISCUSSION THREAD

**Discussion by Nerjaace** [derekantrican/GAS-ICS-Sync#417] [General]
Hi, I have installed the script as per the instructions based on the new interface. I have set the interval to 5
var howFrequent = 5;      
And set up my source calendar
  ["https://app.renterus.com/channelmanager/ical/generate-agente.php?v=1&p=NUVWSlplNHhlV3h6NzZRZ3kwelpSUT09", "ISGAVIO"]
];

I have installed and run the script and it has created a calendar named ISGAVIO and it has populated the events. Since then i have made changes to one of the events at source but the calendar has not updated. Its as if there is no sync every 5mins.

Do you have any suggestions for things i should check?

The Execution log all seems ok
Execution log
2:09:34 PM Notice Execution started
2:09:33 PM Info Intended frequency = 5, Adjusted frequency = 5
2:09:38 PM Notice Execution completed

Thanks in advance.

**Comment by jonas0b1011001** [comment]
Visit [https://script.google.com/home/executions](https://script.google.com/home/executions) and check that the script is actually running every 5 mins. Cclick on one of the executions to see if the log shows any error.

**Reply by Nerjaace** [reply]
Hi Jonas, yes that all seems to be running ok. i have clicked on one of the executions and it is attached
[execution.txt](https://github.com/derekantrican/GAS-ICS-Sync/files/14387223/execution.txt)

There are 5 events currently on the iCal but the actual calendar is still showing 6. I had deleted 1 event earlier today but it has not synced and is still showing on the calendar.

Any ideas?

**Reply by Nerjaace** [reply]
Here i attach a copy of the code.gs so you can see my modification.
[code.txt](https://github.com/derekantrican/GAS-ICS-Sync/files/14387301/code.txt)

**Reply by Nerjaace** [reply]
Hi Jonas, i think i have sorted it.

Thanks for your time :)

**Reply by jonas0b1011001** [reply]
Awesome, let us know if any new issues arise!
